app/services/taxonomy/bulk_update_taxon.rb

```diff
@@ -0,0 +1,27 @@
+module Taxonomy
+  class BulkUpdateTaxon
+    def initialize(root_taxon_content_id)
+      @root_taxon_content_id = root_taxon_content_id
+    end
+
+    def self.call(root_taxon_content_id)
+      new(root_taxon_content_id).bulk_publish
+    end
+
+    def bulk_publish
+      linked_content_item = LinkedContentItemApi.linked_content_item(@root_taxon_content_id)
+      linked_content_item.each do |taxon|
+        PublishTaxonWorker.perform_async(taxon.content_id)
+      end
+    end
+
+    class LinkedContentItemApi
```

Comment:
is there any need for this to be contained in it's own class? I feel this obscures the intention and makes this code harder to read.

For test purposes, wouldn't it be just as easy to stub the response from `GovukTaxonomyHelpers::LinkedContentItem.from_publishing_api` to return your test doubles?